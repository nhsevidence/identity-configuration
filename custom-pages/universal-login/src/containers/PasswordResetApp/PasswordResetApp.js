import React from 'react'
import {Route, Switch} from "react-router";
import Header from "../../components/Header/Header";
import ResetPassword from "../../components/ResetPassword/ResetPassword";
import ResetPasswordSuccess from "../../components/ResetPasswordSuccess/ResetPasswordSuccess";
import NotFound from "../../components/NotFound/NotFound";
import './PasswordResetApp.scss'

class PasswordResetApp extends React.Component {
  render() {
    return (
      <div>
        <div className="wrapper">
          <Header className="col leftCol" />
          <div className="mainContainer col rightCol">
            <Switch>
              <Route exact path="/" component={ResetPassword} />
              <Route exact path='/resetsuccess' component={ResetPasswordSuccess} />
              <Route component={NotFound} />
            </Switch>
          </div>
        </div>
      </div>
    )
  }
}
export default PasswordResetApp
