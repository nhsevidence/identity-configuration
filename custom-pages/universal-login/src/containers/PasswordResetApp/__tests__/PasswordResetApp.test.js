import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import PasswordResetApp from '../PasswordResetApp'

Enzyme.configure({ adapter: new Adapter() });

const props = { history: {}, routes: {}, routerKey: 1 };

describe('LoginApp components', () => {
  it('should render <PasswordResetApp /> correctly', () => {
    expect(shallow(<PasswordResetApp {...props} />)).toMatchSnapshot()
  })
});
