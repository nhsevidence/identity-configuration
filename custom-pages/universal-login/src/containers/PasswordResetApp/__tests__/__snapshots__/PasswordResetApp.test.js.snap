// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginApp components should render <PasswordResetApp /> correctly 1`] = `ShallowWrapper {}`;
